### DISCUSSION TITLE
是否支持显示天赋等级？

### DISCUSSION THREAD

**Discussion by karaikyo** [Arondight/Adachi-BOT#581] [Q&A] [answered]
印象中看过有人发的bot生成的全部角色图片里有角色的天赋等级（当然可能我记错了）。米游社好像是查不到天赋等级的，有没有地方可以我手动输入天赋等级然后bot把图片生成出来呢？

还有个问题是“米游社”命令生成的图片里的头像和名片可以自定义吗，在哪里可以改？

**Comment by Arondight** [comment, marked as answer]
1. 游戏内 ocr 可以办到，好想有个公众号就是这样的，提交 uid ，把角色放到展示第一个，然后游戏内搜角色 ocr 识别（有一定错误率）
2. https://github.com/Arondight/Adachi-BOT/blob/411fb0066c9163eea367f2d5ed9900b90b379e0a/src/views/component/card-8/app.js#L117-L120

**Comment by mark9804** [comment]
> 有没有地方可以我手动输入天赋等级然后bot把图片生成出来呢？

如果想通过手动输入数据的方式实现的话，可以在 `package.js` 里面灌上数据，大概在这个部分（怎么传参也得你自己想办法）

https://github.com/Arondight/Adachi-BOT/blob/411fb0066c9163eea367f2d5ed9900b90b379e0a/src/plugins/package/package.js#L52-L59

然后在这里加一个显示天赋等级的 `div` ，不过你得加上 `v-if` ，除非你能保证每个人都有一份天赋数据

https://github.com/Arondight/Adachi-BOT/blob/411fb0066c9163eea367f2d5ed9900b90b379e0a/src/views/component/card-8/character-box.js#L8-L17

最后改一下 container 的高度

https://github.com/Arondight/Adachi-BOT/blob/411fb0066c9163eea367f2d5ed9900b90b379e0a/src/views/component/card-8/card-8.css#L214

> “米游社”命令生成的图片里的头像和名片可以自定义吗，在哪里可以改？

问题 2 就和楼上一样的回答，你可以通过自行传递参数覆盖掉这段代码逻辑（当然要自己动手写）
